olm/open-liberty-0.0.1.clusterserviceversion.yaml

```diff
@@ -190,19 +190,38 @@ spec:
       kind: OpenLiberty
       name: openliberties.openliberty.io
       resources:
-      - kind: Service
+      - kind: ConfigMap
         name: ""
         version: v1
-      - kind: StatefulSet
+      - kind: Deployment
+        name: ""
+        version: v1
+      - kind: Namespaces
         name: ""
         version: v1
+      - kind: OpenLiberty
```

Comment:
I guess I don't understand why this one is here, since this is the `OpenLiberty` CRD.  Was `scorecard` complaining about this?

Reply:
Basic Tests:
	Status Block Exists: 1/1
	Writing into CRs has an effect: 1/1
	Spec Block Exists: 1/1
OLM Tests:
	Status fields with descriptors: 1/1
	Provided APIs have validation: 0/0
	Owned CRDs have resources listed: 1/1
	CRs have at least 1 example: 1/1
	Spec fields with descriptors: 19/19

Total Score: 88%
SUGGESTION: If it would be helpful to an end-user to understand or troubleshoot your CR, consider adding resources [openliberties/v1alpha1 secrets/v1 serviceaccounts/v1 roles/v1 rolebindings/v1] to the resources section for owned CRD OpenLiberty